src/main/python/powerFactory2json/pf2json.py

```diff
@@ -0,0 +1,3153 @@
+import powerfactory  # @UnresolvedImport @UnusedVariable
+import json
+import inspect
+import os
+
+#################
+# Configuration #
+#################
+
+exported_grid_dir = "../out"
+exported_grid_file = os.path.join(exported_grid_dir, "pfGrid.json")
+
+elements4export = {
+    'nodes': '*.ElmTerm',
+    'lines': '*.ElmLne',
+    'lineTypes': '*.TypLne',
+    'trafos2w': '*.ElmTr2',
+    'trafoTypes2w': '*.TypTr2',
+    'loads': '*.ElmLod',
+    'loadsLV': '*.ElmLodlv',
+    'loadsMV': '*.ElmLodmv',
+    'statGen': '*.ElmGenstat',  # heat pumps, storage, electric vehicles, photovoltaic units ...
+    'extGrid': '*.ElmXNet',
+    'powerPlants': '*.ElmSym',  # renewable power plants
+    'pvs': '*.ElmPvsys',  # additional photovoltaic units
+}
+
+nested_elements4export = {'cpZone'}
+
+
+excluded_fields = {
```

Comment:
any chance to move this into a separate python file and just include it? this would avoid cluttering the whole file with a lot of "useless" information for the actual export